lighthouse-core/report/v2/report-styles.css

```diff
@@ -754,6 +754,17 @@ span.lh-node:hover {
   margin-top: 0;
 }
 
+.lh-run-warnings {
+  font-size: var(--body-font-size);
+  margin: 0;
+  padding: var(--section-padding);
+  border-bottom: 1px solid var(--report-border-color);
+  line-height: var(--header-line-height);
+}
+.lh-run-warnings::before {
+  display: inline-block;
```

Comment:
~interesting, what for?~ oh are you reusing an icon from some other class?

Reply:
> oh are you reusing an icon from some other class?

yeah, the styling from `lh-debug`. Stolen from Eric's #2920